api/src/adapters/search/opensearch_client.py

```diff
@@ -15,10 +15,7 @@
         "default": {
             "type": "custom",
             "filter": ["lowercase", "custom_stemmer"],
-            # Change tokenization to whitespace as the default is very clunky
-            # with a lot of our IDs that have dashes in them.
-            # see: https://opensearch.org/docs/latest/analyzers/tokenizers/index/
-            "tokenizer": "whitespace",
+            "tokenizer": "standard",
```

Comment:
Does this need to match the stemmer chosen in the API request?

Reply:
This is the utils, I found some issues with what I had configured in the prior PR when setting it up with our actual data and fixed it here.